css/src/modal.css

```diff
@@ -80,6 +81,10 @@
 	margin: 0;
 }
 
+.yoast-gutenberg-modal .components-modal__content.has-scrolled-content:not(.hide-header) .components-modal__header {
+	border-bottom-color: #e2e8f0;
+}
```

Comment:
This means there is no border when scrolling. The wanted behavior is always a border/separator.

```suggestion
.yoast-gutenberg-modal .components-modal__content .components-modal__header {
	border-bottom-color: #e2e8f0 !important;
}
```
Unfortunate use of `!important` to get prio, maybe something nicer is achievable.
Kai mentioned that it is fine if our other modals also move to `slate-200`, e.g. Search appearance and social modals?